Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```
@@ -2,39 +2,63 @@
 
 .bocCheckBox
 {
-  margin-right: 0.5em;
+  display: inline-flex;
+  align-items: center;
+}
+
+.bocCheckBox input
+{
+  position: absolute;
+  z-index: -1;
+}
+
+.bocCheckBox span.description
+{
+  display: inline-flex;
```

```suggestion
  display: flex;
```
Since we're in a block-level context (inside the outer flexbox.)

---

@MichaelKetting  As `span` is an inline element, I wanted to keep this property even if it's not necessary here. Setting `flex` instead would make the span a block element. (Or did I miss something?)

---

As you like